### PR TITLE
Added "Afk" effect

### PR DESCRIPTION
**Pull request by OnlyRealNubs**
Makes your camera look to the right. Like how you would prevent yourself from going afk in GTA Online.

### CODE REVIEW

File: ChaosMod/Effects/db/Player/PlayerAFK.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
+/*
+	Effect By OnlyRealNubs
+*/
+
+#include "stdafx.h"
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerAFK.cpp:5]:
```suggestion
#include <stdafx.h>
```

File: ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp [hunk 2]
```diff
@@ -28,7 +28,7 @@ static void OnTick()
 	DISABLE_CONTROL_ACTION(0, 153, true);
 
 	// Disable aiming actions, would cancel forward movement
-	DISABLE_CONTROL_ACTION(0, 25, true); //	INPUT_AIM
+	DISABLE_CONTROL_ACTION(0, 25, true); //	    INPUT_AIM
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp:31]: Redundant diff

**Review by pongo1231 — CHANGES_REQUESTED** [review]
Effect name to AFK rather than Afk imo

File: ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp [hunk 3]
```diff
@@ -42,4 +42,4 @@ REGISTER_EFFECT(nullptr, nullptr, OnTick, EffectInfo
 		.IsTimed = true,
 		.IsShortDuration = true
 	}
-);
\ No newline at end of file
+);
```
Comment (pongo1231) [inline ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp:45]: It's back again 😄

Comment (OnlyRealNubs) [reply ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp:45]: Arghh…. How do I remove this?

Comment (pongo1231) [reply ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp:45]: Are you editing it using the web interface? It seems to automatically add a newline to the end of the file.

Comment (OnlyRealNubs) [reply ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp:45]: Yeah, I do it for small edits where it’s not necessary to open up my ide

Comment (pongo1231) [reply ChaosMod/Effects/db/Player/PlayerKeepRunning.cpp]: Fixed it

**Pull request merged**